mne/channels/tests/test_montage.py

```diff
@@ -443,6 +445,195 @@ def test_read_dig_montage():
         assert_allclose(montage_extra.elp, montage.elp)
 
 
+# XXXX: this should be a function not an alias
+read_polhemus_fast_scan = _read_dig_points
+
+def test_read_dig_montage_using_polhemus_fastscan():
+    """Test FastScan."""
+    N_EEG_CH = 10
+
+    my_electrode_positions = read_polhemus_fast_scan(
+        op.join(kit_dir, 'test_elp.txt')
+    )/1000
+
+    montage = make_dig_montage(
+        # EEG_CH
+        ch_pos=dict(zip(ascii_lowercase[:N_EEG_CH],
+                        np.random.RandomState(0).rand(N_EEG_CH, 3))),
+        # NO NAMED points
+        nasion=my_electrode_positions[0],
+        lpa=my_electrode_positions[1],
+        rpa=my_electrode_positions[2],
+        hpi=my_electrode_positions[3:],
+        hsp=read_polhemus_fast_scan(op.join(kit_dir, 'test_hsp.txt'))/1000,
+        hpi_dev=None,  # XXX: I'm not sure we should allow hpi_dev
+
+        # Other defaults
+        coord_frame='unknown',
+        compute_dev_head_t=False,  # XXX: this one should fall
+        transform_to_head=False,  # XXXX: maybe this one aswell
+    )
+
+    assert montage.__repr__() == (
+        '<DigMontage | '
+        '500 extras (headshape), 5 HPIs, 3 fiducials, 10 channels>'
+    )  # XXX: is this wrong? extra is not in headspace, is it?
```

Comment:
yes but not when reading if we no longer transform. To me this should be assessed from `d['coord_frame']` instead of hardcoded in `__repr__`. This will be addressed in a different PR.